docs/grammar/BallerinaLexer.g4

```diff
@@ -70,6 +70,14 @@ WITH        : 'with' ;
 BIND        : 'bind' ;
 IN          : 'in' ;
 LOCK        : 'lock' ;
+FROM        : 'from' ;
```

Comment:
It is better if we can add a space here, So we have a separation between SQL token and statement tokens.

Reply:
@hasithaa yes.. did the change.